registry/storage/driver/gcs/gcs_test.go

```diff
@@ -69,14 +76,21 @@ func init() {
 		ts = jwtConfig.TokenSource(dcontext.Background())
 	}
 
+	gcs, err := storage.NewClient(context.Background(), option.WithCredentialsJSON(jsonKey))
```

Comment:
we can use the `dContext.Background()`, no need to import the `context` in the gcs.go